package.json

```diff
@@ -262,6 +262,7 @@
     "eslint-plugin-jest": "21.2.0",
     "eslint-plugin-react": "7.1.0",
     "eslint-plugin-wpcalypso": "3.4.1",
+    "find-root": "1.1.0",
```

Comment:
I think this is the problem 😀 

This needs to go into the `dependencies`, not `devDependencies`.

The Docker build only installs production modules, the proceeds to build. We need to consider that this is an application for _building_ Calypso, not just for serving it. Anything required for the actual build should go in `dependencies`.

I ran into this with `uglify-js` and had to move it to `dependencies` as well.

https://github.com/Automattic/wp-calypso/blob/5f15f3365213253724ca0d8798c6cb8160f00bc9/Dockerfile#L33-L43

Reply:
See https://github.com/Automattic/wp-calypso/pull/19759#discussion_r150949913

Reply:
> This needs to go into the `dependencies`, not `devDependencies`.

> The Docker build only installs production modules, the proceeds to build. We need to consider that this is an application for _building_ Calypso, not just for serving it. Anything required for the actual build should go in `dependencies`.

OMG, this seems totally plausible. I wonder why I didn't think of moving it to `dependencies` proper myself.

I might owe you a 🍺 now 😄